src/threading/formatters/JSON.h

```diff
@@ -4,12 +4,6 @@
 
 #include <memory>
 
-#define RAPIDJSON_HAS_STDSTRING 1
-// Remove in v7.1 when removing NullDoubleWriter below and also remove
-// rapidjson include tweaks from  CMake's dynamic_plugin_base target.
```

Comment:
We should now be able to drop all the following ones: Plugins should not need to know about rapidjson anymore.

https://github.com/zeek/zeek/blob/97fa7cdc0a49869ee6605fac9cfc15f11d8c855b/CMakeLists.txt#L343

https://github.com/zeek/zeek/blob/97fa7cdc0a49869ee6605fac9cfc15f11d8c855b/CMakeLists.txt#L351-L354

https://github.com/zeek/zeek/blob/97fa7cdc0a49869ee6605fac9cfc15f11d8c855b/CMakeLists.txt#L1013-L1014

This should make make rapidjson an implementation detail and plugins won't be exposed to it anymore.